tools/getimgversion.sh

```diff
@@ -6,9 +6,9 @@ PRINTALL=0
 
 while [ $# != 0 ]; do
     if [ $1 = "-a" ]; then
-	PRINTALL=1
+    PRINTALL=1
```

Comment:
Why do we loose the identation here?
FWIW this script is useful and used.

Reply:
Good catch, I have no idea why indentation got lost here, each \t should be replaced by four spaces... I will check....

Reply:
@eriknordmark , it turns out that only one tab (\t) was used in the original file, but it seems Github's code viewer is smart enough to put the right indentation... I will fix the indentation...

Reply:
actually if the viewer uses 8 spaces (instead of 4) for each \t then it should look indented, and that's a good example why tabs and spaces shouldn't be mixed in the source code....